### PR TITLE
Parameterize: improve restartability

### PR DESCRIPTION
**Pull request by raimis**
- [x] Fix restartability with non-local queues
- [x] More robust detection of completed vs killed Psi4 jobs
  - [x] Absuse `timer.dat`
  - [x] Update tests

Depend on #519

### CODE REVIEW

**Comment by j3mdamas** [comment]
Is it duplicate of this https://github.com/Acellera/htmd/pull/497? Do we deprecate and close the other PR?

**Comment by j3mdamas** [comment]
Why aren't we sticking with the `htmd.queues.done`, which is general, and relying on the `timer.dat`?

**Comment by raimis** [comment]
`htmd.queues.done` will be present regardless, what happened to the job. `timer.dat` is only present, if Psi4 finished successfully or hit unrecoverable error, while random crashes (e.g. memory shortage), user killings can be recovered.

**Comment by raimis** [comment]
@stefdoerr 

One more PR for your review and merging! :D

File: htmd/qm/psi4.py [hunk 1]
```diff
@@ -118,7 +118,10 @@ def _command(self):
         return 'export HTMD_PSI4_WORKDIR=$(pwd)\npsi4 -i psi4.in -o psi4.out &> psi4.log'
 
     def _completed(self, directory):
-        return os.path.exists(os.path.join(directory, 'psi4.out'))
+        # Abuse "timer.dat" to detect if a Psi4 job has completed.
```
Comment (stefdoerr) [inline htmd/qm/psi4.py]: Is it not possible to use sentinels the same way we use them in our queues? By wrapping the executable in a bash script with the sentinel?

Comment (stefdoerr) [reply htmd/qm/psi4.py]: Ah ok I read the discussion

**Pull request merged**